src/main/java/org/marvelution/jji/tunnel/NgrokConnector.java

```diff
@@ -37,6 +38,7 @@ public NgrokConnector(
         this.details = details;
     }
 
+    @SuppressFBWarnings(value = "NP_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD", justification = "TODO needs triage")
```

Comment:
I don't follow why this needs to be added. The method just checks some values nothing special.
Can you help me understand this?

Reply:
The newer parent POM (needed for 2.479.1 or later) also comes with a newer version of SpotBugs, which is more strict about these errors. Standard practice in Jenkins plugins is to either suppress the violations with this annotation or to disable SpotBugs (or lower its threshold). I usually opt to disable the individual violations in the hopes that they can later be triaged and either fixed or kept as permanent suppressions with reasoning.

Reply:
Thanks for clarifying.